mantle/platform/api/aws/network.go

```diff
@@ -319,6 +319,11 @@ func (a *API) getSubnetID(vpc string) (string, error) {
 	}
 	for _, id := range subIds.Subnets {
 		if id.SubnetId != nil {
+			// Skip `us-east-1c` because m5 instances aren't available there.
```

Comment:
Rather than hard coding a blacklist I'd prefer to see us lookup the type availability of the availability zone and cross reference that versus the requested type.